src/ORM/FieldType/DBDatetime.php

```diff
@@ -48,44 +51,23 @@ class DBDatetime extends DBDate implements TemplateGlobalProvider
      */
     public const ISO_DATETIME_NORMALISED = 'y-MM-dd\'T\'HH:mm:ss';
 
-    /**
-     * Flag idicating if this field is considered immutable
-     * when this is enabled setting the value of this field will return a new field instance
-     * instead updatin the old one
-     */
-    protected bool $immutable = false;
+    private static array $field_validators = [
+        DatetimeFieldValidator::class,
+        // disable parent validator
+        DateFieldValidator::class => null,
+    ];
 
     /**
      * Used to set a specific time for "now", useful for unit tests.
      */
     protected static ?DBDatetime $mock_now = null;
 
-    /**
-     * Set whether this field is mutable (can be modified) or immutable (cannot be modified)
-     */
-    public function setImmutable(bool $immutable): static
-    {
-        $this->immutable = $immutable;
-        return $this;
-    }
-
     public function setValue(mixed $value, null|array|ModelData $record = null, bool $markChanged = true): static
     {
-        if ($this->immutable) {
```

Comment:
DBDatetime was the only DBField that had the 'immutable' ability. I have no idea why and nothing in the codebase called ->setImmutable(). I've simply removed it to be consistent with other DBFields